### ISSUE TITLE
Question: Thread safe writer

### ISSUE THREAD

**Issue by jainabhi1** [RoaringBitmap/RoaringBitmap#600]
Does Roaring bitmap offers thread safe writes ? We are trying to write it into bitmap from multiple threads

**Comment by lemire** [comment]
@jainabhi1 No. We do not have have locks, it is up to the user to handle thread safety.
